src/site/content/en/blog/multi-screen-window-placement/index.md

```diff
@@ -476,6 +462,6 @@ and [Kayce Basques](https://github.com/kaycebasques). Thanks to Laura Torrent Pu
 [cr-dev-twitter]: https://twitter.com/ChromiumDev
```

Comment:
Please update the [blink-component] link to https://bugs.chromium.org/p/chromium/issues/list?q=component:Blink%3EScreen%3EMultiScreen

Reply:
Done

Reply:
Done